src/libraries/System.Private.Xml.Linq/src/System/Xml/Linq/XHashtable.cs

```diff
@@ -238,7 +239,7 @@ public XHashtableState Resize()
             /// Attempt to find "key" in the table.  If the key exists, return the associated value in "value" and
             /// return true.  Otherwise return false.
             /// </summary>
-            public bool TryGetValue(string key, int index, int count, out TValue value)
+            public bool TryGetValue(string key, int index, int count, [MaybeNullWhen(false)] out TValue value)
```

Comment:
Should this be nullable `TValue` with `[NotNullWhen(true)]`?

Reply:
For reference, `Dictionary<TKey, TValue>` and `HashSet<T>` use `MaybeNullWhen(false)` on their `TryGetValue` method. I'm not very sure what are the differences between `NotNullWhen(true)` and `MaybeNullWhen(false)`.

Reply:
perhaps it's generic types, if it's consistent then it's fine